site/source/_posts/2017/11/17-Dojo-2-Beta-4/Dojo-2-Beta-4.md

```diff
@@ -0,0 +1,263 @@
+---
+title: Dojo 2 Beta 4
+date: 2017-11-17 18:00:00
+authorId: kitsonk
+featured_image: featured.png
+---
+
+The Beta 4 Release of Dojo 2 puts us on a clear path to releasing Dojo 2.  It has several significant changes and updates.
+
+{% asset_img featured.png feature-full %}
+
+<!-- more -->
+
+## Getting Beta 4
+
+Beta 4 is currently tagged as `latest` in npm, so all just updating to the latest version will provide you with Beta 4.  We also have a release tag of `beta4` in npm if you want to focus just on using Beta 4.
+
+With Beta 3, we revised our package versions to make it easier to keep a compatible set of packes.  We are releasing all the pre-release versions of Dojo 2 packages under major version `0`, but are introducing breaking changes in the minor release (e.g. `0.2.0` has breaking changes from `0.1.0`).  The versions we publish to npm rely upon a minor range.  You will see that some packages for Beta 4 are `0.3.#` and some are `0.2.#`, but they all depend on a range of version they know they are compatible with.  This means you don't have to rely upon a release tag.  So ideally you would want to ensure if you are managing your dependencies directly, you depend upon a minor release (e.g. `~0.2.0`) for packages.
```

Comment:
but they all depend on a range of version they know they are compatible with -> but all packages depend on a range where compatibility has been verified by the Dojo team.